### DISCUSSION TITLE
Open CV module usage

### DISCUSSION THREAD

**Discussion by fadelfffar** [rainyl/opencv_dart#312] [Q&A] [answered]
Can we just use any module on opencv.org and use it as an function? I'm trying to add the new Face Recognition Module (https://docs.opencv.org/4.x/da/d60/tutorial_face_main.html) with Eigenfaces, Fisherfaces, and Local Binary Patterns Histograms algorithms

**Comment by rainyl** [comment, marked as answer]
No, `cv::fave` is not implemented yet and there is no plan to support this module in this project, all supported modules are listed in https://github.com/rainyl/opencv_dart?tab=readme-ov-file#status , contributions are welcome if you want to do it.

**Reply by fadelfffar** [reply]
Thanks @rainyl keep doing what you doing
